openwisp_utils/admin_theme/static/admin/css/openwisp.css

```diff
@@ -76,6 +76,187 @@ See https://code.djangoproject.com/ticket/33878 */
   --font-family-primary:
     "Roboto", "Lucida Grande", "DejaVu Sans", "Bitstream Vera Sans", Verdana, Arial,
     sans-serif;
+
+  /* ========================================
+     OPENWISP BASE COLOR PALETTE (Tier 1)
+     ======================================== */
+
+  /* Primary Brand Colors */
+  --ow-color-primary: #df5d43;
+  --ow-color-primary-dark: #b31e00;
+  --ow-color-primary-alt: #c74126;
+  --ow-color-primary-light: #ffe5e5;
+  --ow-color-primary-lighter: #fdf2f2;
+
+  /* Success/Positive */
+  --ow-color-success: #498b26;
+
+  /* Gray Scale Spectrum */
```

Comment:
In theory these could change from gray/black to other colors, so what about using some other term, like foreground (abbreviated could be `fg`) or anything else that works well as common denominator?